BranchSDK/BNCPreferenceHelper.m

```diff
@@ -791,6 +798,60 @@ - (void) setInvokeRegisterApp:(BOOL)invoke {
     }
 }
 
+- (BOOL) eeaRegionInitialized {
+    @synchronized(self) {
+        if([self readObjectFromDefaults:BRANCH_PREFS_KEY_DMA_EEA])
+            return true;
+        return false;
```

Comment:
Minor preference to use the ObjC YES/NO over the C true/false. Effectively the same, but more consistent with Apple's style.